File: HelloWorld (Java)/app/src/main/java/ar/edu/itba/example/helloworld/MainActivity.java
package ar.edu.itba.example.helloworld;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import ar.edu.itba.example.helloworld.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        ActivityMainBinding binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
    }
}

File: HelloWorld (Kotlin)/app/src/main/java/ar/edu/itba/example/helloworld/MainActivity.kt
package ar.edu.itba.example.helloworld

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import ar.edu.itba.example.helloworld.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val binding: ActivityMainBinding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
    }
}